docs/topics/functionsquickstart.rst

```diff
@@ -0,0 +1,188 @@
+                    status_code=200
+                )
+
+**function.json**
+    .. code-block:: JSON
+
+        {
+            "scriptFile": "__init__.py",
+            "bindings": [
+                {
+                    "authLevel": "anonymous",
+                    "type": "httpTrigger",
+                    "direction": "in",
+                    "name": "req",
+                    "methods": [
+                        "get",
+                        "post"
+                    ]
+                },
+                {
+                    "type": "http",
+                    "direction": "out",
+                    "name": "$return"
+                }
+            ]
+        }
+
+You can create the zip file yourself using the provided folder structure and file content specified above.
+
+Infrastructure Setup
+====================
+In order to use the ``idem-azurerm`` Function App state module, you must have a resource group for the Function App to
+preside within and a storage account of the kind "Storage" or "StorageV2." The Azure Functions will be stored with that
+storage account and then the Function App will run the file containing all of the functions. Below is a state file that
+sets up the resource group and storage account for this demonstration:
+
+**setup.sls**
+    .. code-block:: yaml
+
+        Ensure resource group exists:
+          azurerm.resource.group.present:
+            - name: "rg-function-app"
+            - location: "eastus"
+            - tags:
+                Organization: "EITR Technologies"
+
+        Ensure storage account exists:
+          azurerm.storage.account.present:
+            - name: "stfunctionapp"
+            - resource_group: "rg-function-app"
+            - location: "eastus"
+            - kind: "StorageV2"
+            - sku: "Standard_LRS"
+            - location: "eastus"
+
+State Module Usage
+==================
+Now that you have deployed the appropriate infrastructure and created the zip file for the Azure Functions, you are
+ready to run the Function App state module. Here are a few important things to note about the module:
+
+* The Function App state module requires a Consumption Plan. If you do not already have a Consumption Plan, the module
+  will create one for you (using a name you specify or a default name). If you do use an existing Consumption Plan, the
+  OS of that plan must match the OS specified within the os_type parameter.
+* The module also allows you to enable Application Insights. Application Insights serve as a monitoring and analytics
+  tool, enabling users to do things like diagnose issues or analyze application usage. If you want to enable Application
+  Insights, you can specify the name of an existing Application Insights Component or one will be created for you (using a
+  name you specify or a default name).
+* The functions_file_path parameter should be the *absolute* path of the .zip file (i.e., "/root/dev/functions.zip").
```

Comment:
put this parameter name and the one in the bullet below in monotype so it stands out